src/main/java/com/epam/vsharstuk/springboot_mvc/config/ServletConfig.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.vsharstuk.springboot_mvc.config;
+
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.support.ResourceBundleMessageSource;
+import org.springframework.web.multipart.MultipartResolver;
+import org.springframework.web.multipart.support.StandardServletMultipartResolver;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.i18n.CookieLocaleResolver;
+import org.springframework.web.servlet.i18n.LocaleChangeInterceptor;
+
+import java.util.Locale;
+
+@Configuration
+//@EnableWebMvc
+public class ServletConfig implements WebMvcConfigurer {
+
+  /*  @Override
```

Comment:
Commented code is a symptom of so called "code smells". Try to avoid commented code. You can use different git branches for it.